### ISSUE TITLE
deluge "download" and "move when completed" directories

### ISSUE THREAD

**Issue by nopoz** [Radarr/Radarr#6451]
It would be nice to have the directory options for the Deluge client that were available in Couchpotato. Two (optional) paths are sent to the deluge client with the torrent so it knows where to download to, and where to move the files when the download is complete. 

Referencing Couchpotato deluge config, you can set: 
- a "download" directory (where the deluge will place files during the download phase)
   - deluge references this in the core module as `download_location` [see deluge docs](https://deluge.readthedocs.io/en/latest/modules/deluge.core.html?highlight=download_location) 
- a "completed directory" (where deluge will move the files once the download phase is complete)
   - deluge references this in the core module as `move_completed` and `move_completed_path` [see deluge docs](https://deluge.readthedocs.io/en/latest/modules/deluge.core.html?highlight=move_completed)

See the Couchpotato source for implementation reference: https://github.com/CouchPotato/CouchPotatoServer/blob/7260c12f72447ddb6f062367c6dfbda03ecd4e9c/couchpotato/core/downloaders/deluge.py

This allows different clients like SickChill, Couchpotato, etc to set downloads to different directories/volumes. This is helpful when your tv and movies are on different volumes. When this is the case, one default download path in deluge doesn't work as well as you would need to copy files across volumes - ie copy from default deluge download directory to another volume which hosts the tv or movie content. 

[AB#1224](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/1224)

**Comment by nopoz** [comment]
This isn't a support request. It's a feature request.

**Comment by fryfrog** [comment]
Duplicate of #2403
